dsa-solutions/lc-solutions/0000-0099/0017-letter-combinations-of-a-phone-number.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: letter-combinations-of-a-phone-number
+id: Letter Combinations of a Phone Number
```

Comment:
replace `id: Letter Combinations of a Phone Number` to `id: letter-combinations-of-a-phone-number`